MNE/disp3D/engine/model/items/network/networktreeitem.cpp

```diff
@@ -238,7 +238,7 @@ void NetworkTreeItem::plotNetwork(const Network& tNetworkData, const QVector3D&
 
         //Plot out edges
         for(int j = 0; j < lNetworkNodes.at(i)->getEdgesOut().size(); ++j) {
-            if(abs(lNetworkNodes.at(i)->getEdgesOut().at(j)->getWeight()) >= vecThreshold.x()) {
+            if(fabs(lNetworkNodes.at(i)->getEdgesOut().at(j)->getWeight()) >= vecThreshold.x()) {
```

Comment:
I found about 20 times where abs() is used for floats. I can go through and change them after this PR if u want.